test/Motion-test.js

```diff
@@ -1,5 +1,5 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
Yes indeed, but render isn't supposed to be static so that's why I did it like this